backend/app/views/spree/admin/products/edit.html.erb

```diff
@@ -4,9 +4,9 @@
       <%= link_to t('spree.new_product'), new_object_url, id: 'admin_new_product', class: 'btn btn-primary' %>
     </li>
   <% end %>
-  <% if defined?(Spree::Frontend::Engine) %>
+  <% if Spree::Backend::Config[:frontend_product_path].call(self, @product) %>
```

Comment:
@waiting-for-dev Do you mean a helper for the `frontend_product_path` like this?

```rb
# backend/app/helpers/spree/admin/products_helper.rb

module Spree
  module Admin
    module ProductsHelper
      def frontend_product_path(product)
        # Not sure about the `self` here, but I think it will work.
        Spree::Backend::Config[:frontend_product_path].call(self, @product) 
      end
    end
  end
end
```

```
<%# backend/app/views/spree/admin/products/edit.html.erb %>

  <% if frontend_product_path(@product) %>
    <li id="view_product_link">
      <%= link_to t('spree.view_product'), frontend_product_path(@product), id: 'admin_view_product', class: 'btn btn-default ml-2' %>
    </li>
  <% end %>
```

Reply:
Yes, I mean that! But I leave it to your discretion :slightly_smiling_face:

Reply:
@waiting-for-dev Cool! I added the helper :)